tests/test_init.py

```
@@ -0,0 +1,27 @@
+from unittest.mock import MagicMock
```

Ruff (0.12.2)

Missing docstring in public module

(D100)